### PR TITLE
Margarethe: Add new theme

### PR DESCRIPTION
**Pull request by MaggieCabrera**
pNEWy-hXf-p2

screenshot

### CODE REVIEW

**Comment by MaggieCabrera** [comment]
This one is ready for review, it should be ready to submit to dotorg

**Comment by henriqueiamarino** [comment]
@MaggieCabrera I checked the following items, and they are all LGTM:
- Templates and elements;
- Skip links;
- Hard-coded lines (with Theme Check)
- Readme file

I found only two topics that can be problematic:

Theme Check alerted us about these three images that can be compacted:
> WARNING: assets/images/margarethe-img-5-scaled.jpg is 590.1 KB in size. Large file sizes have a negative impact on website performance and loading time. Compress images before using them.
> WARNING: assets/images/margarethe-img-4-scaled.jpg is 591.0 KB in size. Large file sizes have a negative impact on website performance and loading time. Compress images before using them.
> WARNING: assets/images/margarethe-img-2-scaled.jpg is 767.5 KB in size. Large file sizes have a negative impact on website performance and loading time. Compress images before using them.

The Theme URI doesn't follow our standards. We have been using showcase pages on the Themeshaper site. So, this theme's page should be:
https://themeshaper.com/margarethe/

File: margarethe/assets/images/margarethe-img-2-scaled.jpg
Comment (alaczek) [inline margarethe/assets/images/margarethe-img-2-scaled.jpg:1]: I think this image will have to be compressed.

File: margarethe/assets/images/margarethe-img-4-scaled.jpg
Comment (alaczek) [inline margarethe/assets/images/margarethe-img-4-scaled.jpg:1]: I think this image will have to be compressed.

File: margarethe/assets/images/margarethe-img-5-scaled.jpg
Comment (alaczek) [inline margarethe/assets/images/margarethe-img-5-scaled.jpg:1]: I think this image will have to be compressed.

**Review by alaczek — DISMISSED** [review]
This looks good to me. The only thing I flagged is the size of some of the images - I don't know what's the recommended size, but I think anything over 350-400KB needs to be compressed.

**Comment by MaggieCabrera** [comment]
I made the changes to the images and the theme URL, I'm bringing this in, thanks for the reviews!

**Pull request merged**